packages/salesforcedx-aura-language-server/.vscode/settings.json

```diff
@@ -0,0 +1,20 @@
+{
+    "files.exclude": {
+      "**/.git": true,
+      "**/.svn": true,
+      "**/.hg": true,
+      "**/CVS": true,
+      "**/.DS_Store": true
+    },
+    "search.exclude": {
+      "**/lib": true,
```

Comment:
I'd like to align these new packages with how the rest of the packages in the mono repo are structured, for consistency and less mental overhead

1. compile src => `out` (js)
1. bundle `out` => `dist`

so not `lib`